src/components/Application/rainbowLegacyStyles.js

```diff
@@ -1,133 +1,5 @@
-    font-style: normal; }
-  
-  @font-face {
-    font-family: 'Lato Bold Italic';
-    src: url(${LatoBoldItalicEot});
-    src: url(${LatoBoldItalicWoff2}) format("woff2"), url(${LatoBoldItalicWoff}) format("woff"), url(${LatoBoldItalicTtf}) format("truetype"), url("${LatoBoldItalicSvg}#Lato-BoldItalic") format("svg"), url("${LatoBoldItalicEot}?#iefix") format("embedded-opentype");
-    font-weight: normal;
-    font-style: normal; }
-  
-  @font-face {
-    font-family: 'Lato Hairline';
-    src: url(${LatoHairlineEot});
-    src: url(${LatoHairlineWoff2}) format("woff2"), url(${LatoHairlineWoff}) format("woff"), url(${LatoHairlineTtf}) format("truetype"), url("${LatoHairlineSvg}#Lato-Hairline") format("svg"), url("${LatoHairlineEot}?#iefix") format("embedded-opentype");
-    font-weight: normal;
-    font-style: normal; }
-  
-  @font-face {
-    font-family: 'Lato Hairline Italic';
-    src: url(${LatoHairlineItalicEot});
-    src: url(${LatoHairlineItalicWoff2}) format("woff2"), url(${LatoHairlineItalicWoff}) format("woff"), url(${LatoHairlineItalicTtf}) format("truetype"), url("${LatoHairlineItalicSvg}#Lato-HairlineItalic") format("svg"), url("${LatoHairlineItalicEot}?#iefix") format("embedded-opentype");
-    font-weight: normal;
-    font-style: normal; }
-  
-  @font-face {
-    font-family: 'Lato Italic';
-    src: url(${LatoItalicEot});
-    src: url(${LatoItalicWoff2}) format("woff2"), url(${LatoItalicWoff}) format("woff"), url(${LatoItalicTtf}) format("truetype"), url("${LatoItalicSvg}#Lato-Italic") format("svg"), url("${LatoItalicEot}?#iefix") format("embedded-opentype");
-    font-weight: normal;
-    font-style: normal; }
-  
-  @font-face {
-    font-family: 'Lato Light';
-    src: url(${LatoLightEot});
-    src: url(${LatoLightWoff2}) format("woff2"), url(${LatoLightWoff}) format("woff"), url(${LatoLightTtf}) format("truetype"), url("${LatoLightSvg}#Lato-Light") format("svg"), url("${LatoLightEot}?#iefix") format("embedded-opentype");
-    font-weight: normal;
-    font-style: normal; }
-  
-  @font-face {
-    font-family: 'Lato Light Italic';
-    src: url(${LatoLightItalicEot});
-    src: url(${LatoLightItalicWoff2}) format("woff2"), url(${LatoLightItalicWoff}) format("woff"), url(${LatoLightItalicTtf}) format("truetype"), url("${LatoLightItalicSvg}#Lato-LightItalic") format("svg"), url("${LatoLightItalicEot}?#iefix") format("embedded-opentype");
-    font-weight: normal;
-    font-style: normal; }
-  
-  @font-face {
-    font-family: 'Lato';
-    src: url(${LatoRegularEot});
-    src: url(${LatoRegularWoff2}) format("woff2"), url(${LatoRegularWoff}) format("woff"), url(${LatoRegularTtf}) format("truetype"), url("${LatoRegularSvg}#Lato-Regular") format("svg"), url("${LatoRegularEot}?#iefix") format("embedded-opentype");
-    font-weight: normal;
-    font-style: normal; }
+  @import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap');
```

Comment:
Here we need to be careful since we are using these font-faces defined and it can stop working since they can have other names.
For example, users can be used in their code: ```font-family: 'Lato Bold Italic'```

Reply:
i just included all Lato available fonts in google-fonts... using google-font source url for fonts i don't think is a good idea as it can change in time... lets see if there is a way to add an alias

Reply:
there is no way to add an alias to web fonts, only to local
sol 1: private font hosting
sol 2:  what if instead of using a direct url to the font  asset.. we set the url in base64.. like in this page https://www.filamentgroup.com/lab/font-loading.html... it should increase the size of the lib if we include all font-assets but we can include only the most supported like ttf, woff, woff2...